pontoon/search/templates/search/widgets/entity_list.html

```diff
@@ -0,0 +1,31 @@
+
+{% macro list_entities(entities, entities_url, has_more) %}
+<ul class="entity-list" data-has-more="{{ 'true' if has_more else 'false' }}">
+  {% for entity in entities %}
+    <li class="entity-item">
+        <span>{{ entity.string }}</span>
+        <span>{{ entity.key }}</span>
```

Comment:
We should not render entity keys where they aren't meaningful. We should drop them at least for PO entities, because they are essentially the same as the source string.